osu.Game.Rulesets.Mania/Timing/TimingChange.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+namespace osu.Game.Rulesets.Mania.Timing
+{
+    public class TimingChange
+    {
+        /// <summary>
+        /// The time at which this timing change happened.
+        /// </summary>
+        public double Time;
+
+        /// <summary>
+        /// The beat length.
+        /// </summary>
+        public double BeatLength = 500;
+
+        /// <summary>
+        /// The speed multiplier.
+        /// </summary>
+        public double SpeedMultiplier = 1;

```

Comment:
Wouldn't worry too much about this for now since it's mania specific and this isn't a mania branch, though. This is just necessary to bring mania to life again.